pkg/smokescreen/conntrack/instrumented_conn_test.go

```diff
@@ -0,0 +1,95 @@
+package conntrack
+
+import (
+	"io"
+	"net"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestInstrumentedConnByteCounting sends a fixed size message between two
+// instrumented connections and ensures the bytes in and out are correct.
+func TestInstrumentedConnByteCounting(t *testing.T) {
+	assert := assert.New(t)
+
+	addr := "localhost:8888"
```

Comment:
Could we listen on a dynamic available port here instead of hardcoding this?

Reply:
👍